tests/installation/bootloader_s390.pm

```diff
@@ -98,6 +98,10 @@ EO_frickin_boot_parms
     my $max_retries = 7;
     for (1 .. $max_retries) {
         eval {
+            # Define memory to behave the same way as other archs
+            # and to have the same configuration through all s390 guests
+            $s3270->sequence_3270("String(\"DEFINE STORAGE 1G\")", "ENTER",);
```

Comment:
can you actually use the same value as for other arch's? e.g. use the same variable and convert MB to GB instead of hardcoding?